_posts/2020-06-22-new-web-site.md

```diff
@@ -0,0 +1,40 @@
+---
+title:  "New ROOT Web Site!"
+layout: archive
+author: Olivier Couet
+---
+
+If you are reading this blog post it means you are surfing on the New ROOT Web Site!
+
+This new web site is a complete redesign, both from the content point of view and on
+the technology used to build it.
```

Comment:
`both from the content point of view and on the technology used to build it` -> `in terms of content as well as the technology behind it`